src/features/vault/components/PoolSummary/LabeledStat/styles.js

```diff
@@ -5,6 +5,11 @@ const styles = (theme) => ({
     alignItems: props => (props.align ? props.align : 'center'),
     justifyContent: 'center',
   },
+  container2: {
```

Comment:
Can we use a more descriptive class name, related to the functionality it represents?